### ISSUE TITLE
Additional log message for rclone cleanup

### ISSUE THREAD

**Issue by ctrlnot** [rclone/rclone#4915]
I just saw the last suggestion on my solved [clean up issue](https://forum.rclone.org/t/cleanup-does-nothing/18545/3) from the forums. A message saying that the clean up would take a while will be good.

**Comment by ncw** [comment]
How does this look?

```
$ rclone cleanup TestDrive:
2021/01/09 13:13:27 NOTICE: Google drive root '': Note that emptying the trash happens in the background and can take some time.
```

I made a beta with that in here [v1.54.0-beta.5050.f7586171c.fix-4915-drive-trash](https://beta.rclone.org/branch/fix-4915-drive-trash/v1.54.0-beta.5050.f7586171c.fix-4915-drive-trash/) on branch [fix-4915-drive-trash](https://github.com/rclone/rclone/tree/fix-4915-drive-trash) (uploaded in 15-30 mins)

**Comment by ctrlnot** [comment]
Looks good for me. Thanks!

**Comment by ncw** [comment]
I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.54
